utilities/balancing-position-control/README.md

```diff
@@ -0,0 +1,26 @@
+# balancing-position-control
+The **balancing-position-control** is an application that allows a humanoid robot to move the center-of-mass (CoM) by
+following a given trajectory.
+
+## 🏃 How to use the application
+The fastest way to use the utility is to run the `python` application
+[`blf-balancing-position-control.py`](./script/blf-balancing-position-control.py).
+If you correctly installed the framework, you can run the application from any folder.
```

Comment:
I would suggest adding some more detail on how to run the app, and on the assumptions you have about the robot status. For example, does the robot need to be on the ground? Is the com supposed to be close the first know point? Are the feet kept fixed?